readme.txt

```diff
@@ -1,8 +1,22 @@
+=====BASIC SET-UP =====
+* Create .env file; the contents are described in ENVIRONMENT VARIABLES
+* Create a certificate and key. Place those in cert/cert/ (by default)
```

Comment:
I provided the console commands in the next section. I could still provide a link if you want.